### PR TITLE
fixed typos and closed open brackets (#807)

### PR DESCRIPTION
**Pull request by iHarshit15**
This change is [Reviewable](https://reviewable.io/reviews/google/openhtf/1126)

### CODE REVIEW

File: openhtf/output/callbacks/mfg_inspector.py [hunk 1]
```diff
@@ -93,15 +93,15 @@ class MfgInspector(object):
...
+  **Important** the conversion of the TestRecord to protobuf as specified in
   the _converter callable attribute only occurs once and the resulting protobuf
   is cached in memory on the instance.
 
   The upload callback will upload to mfg-inspector.com using the given
   username and authentication key (which should be the key data itself, not a
   filename or file).
 
-  In typical productin setups, we *first* save the protobuf to disk then attempt
+  In typical production setups, we *first* save the protobuf to disk then attempt
```
Comment (dbhatman) [inline openhtf/output/callbacks/mfg_inspector.py:104]: Would you mind tweaking this to get every line to fix into 80 characters?

**Review by dbhatman — COMMENTED** [review]
Thank you for these corrections. Everything looks good except the one comment I left.

We try to employ an 80 character line limit and that one line is now past 80 characters.

**Comment by iHarshit15** [comment]
Sure, can we write it as "prodn" or should I leave it as it was?

File: openhtf/output/callbacks/mfg_inspector.py (hunk 1 above)
Comment (iHarshit15) [reply openhtf/output/callbacks/mfg_inspector.py:104]: Sure, can we write it as "prodn" or should I leave it as it was?

Comment (iHarshit15) [reply openhtf/output/callbacks/mfg_inspector.py]: Would you review the change I made addressing to the comment?

Comment (dbhatman) [reply openhtf/output/callbacks/mfg_inspector.py]: Ah sorry I should have been more clear. Lets move 1 word down one line at a time here. Basically what I mean is change this paragraph to be:


```
  In typical production setups, we *first* save the protobuf to disk then
  attempt to upload the protobuf to mfg-inspector. In the event of a network,
  outage the result of the test run is available on disk and a separate process
  can retry the upload when the network is available.
```

Comment (iHarshit15) [reply openhtf/output/callbacks/mfg_inspector.py]: Thank You I changed it as desired

**Review by dbhatman — APPROVED** [review]
Thank you!

**Pull request merged**